scripts/validate-env.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+# Script to validate that the environment variables are configured correctly
+
+# Ensure script is always run from the root of the repository
+cd $(dirname $0)/..
+
+extract_var () {
+    FILE=$1
+    VARNAME=$2
+    awk -F '=' "/${VARNAME}/ "'{print $2}' "${FILE}"
+}
+
+validate_var() {
+    VAR=$(extract_var $1 $2)
+    echo -n "Validating ${2}: "
+    $3 $VAR
+}
+
+ERRORS=
+
+is_dir() {
+    if [ -d $1 ]; then
+        echo "$1 exists"
+    else
+        echo "${1} is NOT a valid directory"
+        ERRORS="yes"
+    fi
+}
+
+url_with_schema() {
+    regex='https?://.*'
```

Comment:
this should be `http` 😬

Reply:
`s?` would mean that `s` either appears, or not.